.github/workflows/tests.yaml

```diff
@@ -76,6 +86,98 @@ jobs:
                   path: |
                       .environment/gn_out/.ninja_log
                       .environment/pigweed-venv/*.log
+            - name: Validate that xml are parseable
+              timeout-minutes: 10
+              # The sub-items being run here are the same as the input XMLs listed
+              # at src/app/zap-templates/zcl/zcl.json
+              #
+              # This ensures that the syntax of the XML can always be parsed/validated, however it
+              # does not enforce that the content is understood (that part is covered by parser
+              # unit tests)
+              #
+              # Also note ordering: global attributes are parsed first because other XML files may
+              # reference these. 
+              #
+              # TODO: some multipass/postprocess may be more appropriate to allow any ordering of 
```

Comment:
Please file an issue and reference here with `TODO(#issue):`

Reply:
This one I was unsure about filing issues: is there value in such an issue?

Reply:
Fair enough